src/prayer/PrayerSingle/index.js

```diff
@@ -39,10 +74,13 @@ const PrayerSingle = memo(
     prayer,
     action,
     isLoading,
+    navigation,
     ...props
   }) => (
-    <View>
-      {!isLoading && <AbsolutePositionedView>{action}</AbsolutePositionedView>}
+    <Wrapper navigation={navigation}>
+      {!isLoading && !getNotification(navigation) ? (
```

Comment:
```suggestion
      {!isLoading && navigation ? (
```

Reply:
So you're saying that just the existence of the `navigation` prop at all ... regardless of the value of the `notification` param, should be enough to assume it's coming from a notification? Is that just because we don't have anything else that might be coming through on a `navigation` prop? 

Just want to make sure I understand the thought process.

Reply:
yea that's what I was thinking. In all other cases, it will be mounted as a component in a tree.

Reply:
It looks like `navigation` gets passed down regardless. I did a `console.log` of it just running the prayer module normally and I get this. So I'm not sure I can really get rid of that function. We're going to have to look for something more than just that the prop doesn't exist.

![Screen Shot 2020-07-27 at 11 27 01 AM](https://user-images.githubusercontent.com/3229463/88560417-1a68b780-cffc-11ea-9f1f-d3904c00db3e.png)

Reply:
Hey @redreceipt Can you please take a look at this one again based on the information above when you get a chance? I pretty much put it back the way it was before attempting to get it to just use the existence of the `navigation` prop.